src/main/java/com/flightstats/hub/spoke/SpokeReadContentDao.java

```diff
@@ -0,0 +1,167 @@
+package com.flightstats.hub.spoke;
+
+import com.flightstats.hub.cluster.Cluster;
+import com.flightstats.hub.dao.ContentDao;
+import com.flightstats.hub.dao.QueryResult;
+import com.flightstats.hub.exception.ContentTooLargeException;
+import com.flightstats.hub.exception.FailedQueryException;
+import com.flightstats.hub.exception.FailedWriteException;
+import com.flightstats.hub.metrics.ActiveTraces;
+import com.flightstats.hub.metrics.Traces;
+import com.flightstats.hub.model.*;
+import com.google.common.base.Optional;
+import org.apache.commons.lang3.NotImplementedException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Inject;
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectOutputStream;
+import java.util.Collections;
+import java.util.List;
+import java.util.SortedSet;
+import java.util.TreeSet;
+
+public class SpokeReadContentDao implements ContentDao {
+
+    private final static Logger logger = LoggerFactory.getLogger(SpokeReadContentDao.class);
+
+    @Inject
+    private RemoteSpokeStore spokeStore;
+
+    @Override
+    public ContentKey insert(String channelName, Content content) throws Exception {
+        throw new NotImplementedException("SpokeReadContentDao.insert not implemented");
+    }
+
+    @Override
+    public SortedSet<ContentKey> insert(BulkContent bulkContent) throws Exception {
```

Comment:
this is nearly the same as SpokeWriteContentDao.insert

Reply:
Yeah, I thought about making a SpokeContentUtil class that does common Spoke stuff but by the time I got to that point I was trying to get this across the finish line and limit scope creep :)

I'm going to add an issue for this as well.

Reply:
See #970